v_4_8_0/files/k8s-resource/calico-all.yaml

```diff
@@ -42,17 +42,17 @@ data:
   cni_network_config: |-
     {
       "name": "k8s-pod-network",
-      "cniVersion": "0.3.0",
+      "cniVersion": "0.3.1",
       "plugins": [
         {
           "type": "calico",
           "log_level": "info",
+          "nodename": "__KUBERNETES_NODE_NAME__",
```

Comment:
Why move this from alphabetical ordering?

Reply:
Main goal was to keep it as close to https://docs.projectcalico.org/v3.9/manifests/calico.yaml as possible. That way we can compare changes in newer versions using `diff`.